docs/index.md

```diff
@@ -60,6 +59,31 @@ Try it here!
    print(eigen_result$vectors)
 ```
 
+```{eval-rst}
+.. replite::
+   :kernel: xcpp
+   :height: 600px
+
+   #include <stdio.h>
+   #include <math.h>
+
+   void funky_sin_wave(int length) {
+      for (int y = 0; y < 20; y++) {
+         for (int x = 0; x < length; x++) {
+               double wave = sin(x * 0.1);
+               if ((int)(10 + 10 * wave) == y) {
+                  printf("*");
+               } else {
+                  printf(" ");
+               }
+         }
+         printf("\n");
+      }
+   }
+
+   funky_sin_wave(100);
+```
```

Comment:
I ran out of ideas hence simply added a sine wave function 😅 
Hope that is okay !

Reply:
Should look something like this 

![image](https://github.com/user-attachments/assets/0fb8d07a-ca95-4e1d-a970-2391cd89a3c9)